backend/utils/db_utils.py

```
@@ -0,0 +1,61 @@
+from datetime import datetime
+from models import Client, ClientReservations, Reservation
+from sqlalchemy.exc import SQLAlchemyError
```

Ruff (0.8.2)

`sqlalchemy.exc.SQLAlchemyError` imported but unused

Remove unused import: `sqlalchemy.exc.SQLAlchemyError`

(F401)